sample/src/main/java/com/chuckerteam/chucker/sample/HttpBinClient.kt

```diff
@@ -127,6 +128,24 @@ class HttpBinClient(
         )
     }
 
+    private fun fakeGzipRequest() {
```

Comment:
We can avoid custom function with a header on a Retrofit function. We could also use `/post` instead of `/gizp` to get back the result instead of `405`.

```kotlin
@POST("/post")
@Headers("Content-Encoding: gzip")
fun postGzipped(@Body body: Data): Call<Any?>
```

Reply:
Yeah, this one looks nicer. Added `/gzip` just to have a visual clue on what request it is.